src/commands/cmd_set.cc

```diff
@@ -371,6 +418,7 @@ REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandSAdd>("sadd", -3, "write", 1, 1, 1),
                         MakeCmdAttr<CommandSDiff>("sdiff", -2, "read-only", 1, -1, 1),
                         MakeCmdAttr<CommandSUnion>("sunion", -2, "read-only", 1, -1, 1),
                         MakeCmdAttr<CommandSInter>("sinter", -2, "read-only", 1, -1, 1),
+                        MakeCmdAttr<CommandSInterCard>("sintercard", -3, "read-only", 1, -1, 1),
```

Comment:
@PragmaTwice Would it be better to use this change on the syntax `SINTERCARD numkeys key [key ...] [LIMIT limit]` ? It seems this range is related to `watch`
```suggestion
                        MakeCmdAttr<CommandSInterCard>("sintercard", -3, "read-only", 2, -1, 1),
```

Reply:
The best solution here is write a `CommandKeyRangeGen` function, since we have to determine the last position of the key sequence.

Reply:
Thanks. I've got it.